include/boost/process/v2/ext/impl/cmd.ipp

```diff
@@ -372,10 +371,10 @@ shell cmd(boost::process::v2::pid_type pid, boost::system::error_code & ec)
 
     std::unique_ptr<kvm_t, closer> kd{kvm_openfiles(nullptr, nullptr, nullptr, KVM_NO_FILES, nullptr)};
 
-    if (!boost::process::v2::detail::ext::kd) {ec = detail::get_last_error(); return vec;}
-    if ((proc_info = kvm_getproc2(boost::process::v2::detail::ext::kd, KERN_PROC_PID, pid, sizeof(struct kinfo_proc2), &cntp))) 
+    if (!kd.get()) {ec = detail::get_last_error(); return vec;}
+    if ((proc_info = kvm_getproc2(kd.get().get().get().get().get().get().get(), KERN_PROC_PID, pid, sizeof(struct kinfo_proc2), &cntp))) 
```

Comment:
that ain't looking right

Reply:
That's because I am out to `.get()` you! >:D